app/shared/components/section-title/SectionTitle.styles.ts

```diff
@@ -0,0 +1,30 @@
+import { AppTypography } from '~/constants';
+
+export const imageSizes = (isLaptopAndAbove: boolean) => {
+  return isLaptopAndAbove
+    ? {
+        width: 32,
+        height: 30
+      }
+    : {
+        width: 22,
+        height: 20
+      };
```

Comment:
```suggestion
  return isLaptopAndAbove ? { width: 32, height: 30 } : { width: 22, height: 20 };
```


Make this change and you will pass ESLint checks
I do not know why ESLint is currently editing ternary operators incorrectly, so I will investigate this issue later, but for now, write it in single line

Reply:
Ok, thanks for help. Was wondering how to fix that.